### PR TITLE
Feature disabling flags & some other stuff

### PR DESCRIPTION
**Pull request by UncleClapton**
* Buncha readme updates
    * Add some troubleshooting notes
    * Add documentation about `false` behavior
    * Add complete default options object.
* When `contentSecurityPolicy` is `false`, `buildCSPHeaders()` now returns an empty array.
* Fixed a bug in `buildCSPHeaders()` that would would throw a reference error if `connect-src`, `script-src`, or `style-src` were configured to `false` in dev mode.
* When `permissionsPolicy` is `false`, `buildPermissionsPolicyHeaders` now returns an empty array.
* Specific directives may be removed from `Permissions-Policy` entirely by setting the directive to `false` in the `permissionsPolicy` object.
    * This was done to make it consistent with `contentSecurityPolicy`
* All other security headers may be disabled by setting their respective settings to `false`

### CODE REVIEW

File: lib/nextSafe.js [hunk 1]
```diff
@@ -3,7 +3,16 @@ const buildCSPHeaders = require('./buildCSPHeaders.js')
 const buildPermissionsPolicyHeaders = require('./buildPermissionsPolicyHeaders.js')
 
 
+function hObj(key, value, defaultValue) {
```
Comment (trezy) [inline lib/nextSafe.js:6]: Can we give this a more expressive name? I'm all about abstracting functions to remove duplication, but I want to make sure that those functions are clearly names.

**Review by trezy — CHANGES_REQUESTED** [review]
This looks great other than renaming the `hObj` function to be more clear about it's purpose. Great work so far! 🥳

File: lib/nextSafe.js (hunk 1 above)
Comment (UncleClapton) [reply lib/nextSafe.js:6]: says the person who named the CSP directive resolver `v`

**Comment by UncleClapton** [comment]
@trezy Since there's gonna be more work required to get types working correctly, how about we separate that into a different PR so this can be merged in.

**Comment by UncleClapton** [comment]
I also added next 10 and 11 to the peer dependencies. Used the latest known secure versions for each.

File: lib/models/CSP.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (trezy) [inline lib/models/CSP.js:59]: That's... a lot of newlines. 🤔

**Review by trezy — APPROVED** [review]
If you wouldn't mind clearing the I-fell-asleep-on-my-keyboard amount of newlines in the CSP model, that'd be amazing. Otherwise, LGTM! 👍🏻

File: lib/models/CSP.js (hunk 2 above)
Comment (UncleClapton) [reply lib/models/CSP.js:59]: daheck how did that happen

**Comment by UncleClapton** [comment]
@trezy all fixed

**Pull request merged**